src/collective/lazysizes/profiles/uninstall/registry.xml

```diff
@@ -1,4 +1,13 @@
 <?xml version="1.0"?>
 <registry>
+  <!-- settings -->
   <records interface="collective.lazysizes.interfaces.ILazySizesSettings" remove="true" />
+  <!-- Plone 5 - resources -->
+  <records condition="have plone-5" remove="True" prefix="plone.bundles/lazysizes" interface="Products.CMFPlone.interfaces.IBundleRegistry"/>
+  <records condition="have plone-5" remove="True" prefix="plone.resources/lazysizes" interface='Products.CMFPlone.interfaces.IResourceRegistry'/>
+  <records condition="have plone-5" remove="True" prefix="plone.resources/lazysizes-twitter" interface='Products.CMFPlone.interfaces.IResourceRegistry'/>
+  <!-- Plone 5 - plone bundle does not depend on any other per default -->
+  <records condition="have plone-5" prefix="plone.bundles/plone" interface="Products.CMFPlone.interfaces.IBundleRegistry" purge="False">
+    <value key="depends"></value>
```

Comment:
are you sure this is right? what happens if there are other add-ons installed?

Reply:
Yes, that might be a problem.
We can also leave out changing the plone bundle's dependency while installing in the first place - and leave it up to the integrator to do this little optimization. I did not do any tests but I think it's better to have lazysizes loaded as first javascript, so it can do it's works as soon as possible.